TEST_ENFORCEMENT.sh

```
@@ -0,0 +1,77 @@
+  echo "✅ PASS: console.log detected by Biome"
+else
+  echo "❌ FAIL: console.log not detected"
+fi
+rm -f /tmp/test-console.ts
+echo ""
+
+# Test 2: Package.json in packages
+echo "Test 2: Trying to create package.json in packages/..."
+mkdir -p /tmp/test-packages/logger
+echo '{}' > /tmp/test-packages/logger/package.json
+cd /tmp/test-packages
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)